debian/cloudstack-agent.install

```diff
@@ -18,12 +18,11 @@
 /etc/cloudstack/agent/agent.properties
 /etc/cloudstack/agent/environment.properties
 /etc/cloudstack/agent/log4j-cloud.xml
+/etc/default/cloudstack-agent
 /etc/profile.d/cloudstack-agent-profile.sh
 /etc/logrotate.d/cloudstack-agent
-/etc/init.d/cloudstack-agent
 /usr/bin/cloudstack-setup-agent
```

Comment:
@wido can we also move the debian directory to packaging folder? The deb files are build at `..` which is the parent folder of the cloudstack repo/source code, and causes issues when doing concurrent builds using Jenkins.